packages/wrangler/src/dev/dev.tsx

```diff
@@ -171,25 +171,39 @@ export function DevImplementation(props: DevProps): JSX.Element {
 	);
 }
 
+let ip: string;
```

Comment:
Sheer madness, I know.

Reply:
The problem is that `useHotkeys()` relies on these values for it's `b` "open a browser" feature. If we store this in proper React state, it gets the value fine, but we then need to call `setIp` or whatever in the `onReady` callback. This causes the entire component to re-render, which re-initializes Miniflare, which causes the `onReady` callback to be called again, etc. etc.

I tried an `if (newValue !== newValue) { setValue() }`, but still couldn't prevent a render loop.